src/bootstrap/step.rs

```diff
@@ -789,6 +786,10 @@ pub fn build_rules<'a>(build: &'a Build) -> Rules {
          .dep(move |s| s.name("tool-build-manifest").target(&build.config.build).stage(0))
          .run(move |_| dist::hash_and_sign(build));
 
+    rules.install("install", "path/to/nowhere")
+         .dep(|s| s.name("default:dist"))
```

Comment:
Oh I was thinking that this is where we could apply the fix, not depending on `default:dist`? Ideally the `install` target doesn't depend on *any* source tarball, so that stage is just skipped entirely. Does that make sense?